install-with-linux.md

```diff
@@ -316,7 +326,9 @@ The installation will report that ChromeOS was installed when it is finished. Be
 
 17. After saving, commit the new entries to Grub.
 
-```sudo update-grub```
+```sudo update-grub``` <br/> 
+  or if that dosent work, <br/>
```

Comment:
This should mention that Arch Linux users (or any non-Debian/Ubuntu distro) should use the bottom one, as update-grub is a Debian/Ubuntu script. Also, doesn't is misspelled.